packages/algob/src/builtin-tasks/unbox-template.ts

```diff
@@ -0,0 +1,24 @@
+import { unbox } from "../internal/cli/unbox-template";
+import { task } from "../internal/core/config/config-env";
+import { TASK_UNBOX_TEMPLATE } from "./task-names";
+
+export default function (): void {
+  task(TASK_UNBOX_TEMPLATE, "Unboxes a new dapp template using algorand-builder")
+    .addFlag(
+      "force",
+      "Unbox project in the current directory regardless of its " +
+      "state. Be careful, this\n                will overwrite files " +
```

Comment:
do we need the spaces here ?

Reply:
Yes, for displaying help. 
![image](https://user-images.githubusercontent.com/33264364/111604572-77b7b000-87fb-11eb-85b6-2245d5d06c85.png)

Reply:
ok